pages/genre/genre.go

```diff
@@ -42,3 +45,22 @@ func containsLowerCase(array []string, search string) bool {
 
 	return false
 }
+
+func averageGenreScore(user *arn.User, animes []*arn.Anime) string {
+	if user == nil {
+		return ""
+	}
+
+	counter := 0.0
+	scores := 0.0
+
+	for _, anime := range animes {
+
```

Comment:
This newline has no meaning, it should be deleted.

Reply:
Ah sorry forgot to follow code style. Was so late. Might be that I can't make new pull requests till evening.